app.js

```diff
@@ -136,7 +137,7 @@ function repopulateUserQueue() {
 
 app.use(function onError(err, req, res, next) {
   const errorId = sentry.captureException(err);
-
+  console.log(err);
```

Comment:
@LearningNerd Is this intentional?

Reply:
Yep, do you think I should remove it? I added this because a syntax error caused a 500 error when the tests ran, but it didn't show a stack trace in the test runner.

Reply:
@LearningNerd If we keep it, it should be `console.error`